src/components/ReviewConfirm/DocumentDelivery.vue

```diff
@@ -30,6 +30,6 @@ import { BusinessContactIF } from '@/interfaces'
 @Component({})
 export default class DocumentDelivery extends Vue {
   @Getter getBusinessContact!: BusinessContactIF
-  @Getter getCompletingPartyEmail!: string
+  @Getter getUserEmail!: string
```

Comment:
This is just the user's email, until we do something with it.

Reply:
Is still the account email but just a property rename?

Reply:
Yes. "Account email" would be another good name for it. My point is, it's not the completing party email until we say it is.

Reply:
oh for sure, account / user etc, allll the same to me.
I follow now!